perllib/FixMyStreet/Cobrand/CentralBedfordshire.pm

```diff
@@ -0,0 +1,24 @@
+package FixMyStreet::Cobrand::CentralBedfordshire;
+use parent 'FixMyStreet::Cobrand::Whitelabel';
+
+use strict;
+use warnings;
+
+sub council_area_id { return 21070; }
+sub council_area { return 'Central Bedfordshire'; }
+sub council_name { return 'Central Bedfordshire Council'; }
+sub council_url { return 'centralbedfordshire'; }
+sub send_questionnaires { 0 }
+
+sub disambiguate_location {
+    my $self    = shift;
+    my $string  = shift;
+
+    return {
+        %{ $self->SUPER::disambiguate_location() },
+        centre => '52.006697,-0.436005',
+        bounds => [ 51.805087, -0.702181, 52.190913, -0.143957 ],
```

Comment:
There's also `bin/fixmystreet.com/generate_council_location` to save you from having to look these values up manually.

Reply:
😲